compiler/luci/import/src/Nodes/CircleConst.cpp

```diff
@@ -160,7 +164,45 @@ CircleNode *CircleConstNodeBuilder::build(TensorIndex tensor_index,
   const auto c_buffer = const_tensor->buffer();
   const auto r_buffer = r_buffers[c_buffer];
   assert(r_buffer != nullptr);
-  const auto buffer = wrap(r_buffer->data());
+  // temporary buffer to provide raw data from file
+  // must have life time same or longer than 'buffer' variable
+  std::vector<uint8_t> temp_buffer;
+  // const auto buffer = wrap(r_buffer->data());
+  luci::VectorWrapper<uint8_t> buffer(nullptr);
+  if (r_buffer->offset() > 1)
```

Comment:
Q) Is it ok to use `r_buffer->data()` for offset == 1?

Reply:
no. `1` for size and offset is invalid.
I'll add some checks here :)

Reply:
I thought that condition might be `if (r_buffer->offset() > 0)`.

Reply:
`offset`, `size` with `1` is made for initial values for the extended Buffer.
these value must be updated to real offset and size after the full stream of file data is generated, that is we'll do something like `seek(offset)` and `read(size)` later in import.